src/views/data/ControlBar.js

```diff
@@ -1,89 +1,79 @@
 import React from 'react';
-import {connect} from 'react-redux';
-import {bindActionCreators} from 'redux';
-import _ from 'lodash';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
 import timeout from 'util/timeout';
+import _ from 'lodash';
 
 import {
     getTableSearch,
-    setTableSearch
+    setTableSearch,
+    getTableFilter,
 } from 'core/reducer';
 
-import {useModals} from 'util/ModalProvider';
+import API from 'core/api';
+
+import { useModals } from 'util/ModalProvider';
 import FilterDataModal from './FilterDataModal';
-import {message, Input, Button, Menu, Dropdown, Icon} from 'antd';
+import { message, Input, Button, Menu, Dropdown, Icon } from 'antd';
 
 import './ControlBar.scss';
 
 // TODO: Pass data here...
-const exportCsv = async (data) => {
+const exportCSV = async (filter, search, dataIsFiltered) => {
```

Comment:
I think this whole function could be reduced quite a bit since we're not preparing the export client side anymore. I added a bunch of fluff originally since we were going to format it client side but most isn't needed anymore. E.g.,
```JavaScript
const exportCSV = (filter, search) => {
    let query;
    if (filter || search) {
        query = { filter, search };
    }
    window.open(API.getExportUrl(query), '_blank');
}
````

Reply:
implemented your suggested changed.